test/src/test/java/hudson/model/ComputerSetTest.java

```diff
@@ -76,4 +82,15 @@ public void nodeOfflineCli() throws Exception {
             cli.close();
         }
     }
+
+    @Test
+    public void getComputerNames() throws Exception {
+        assertThat(ComputerSet.getComputerNames(), is(empty()));
+        j.createSlave("aNode", "", null);
+        assertThat(ComputerSet.getComputerNames(), hasSize(1));
+        assertThat(ComputerSet.getComputerNames(), contains("aNode"));
+        j.createSlave("anAnotherNode", "", null);
+        assertThat(ComputerSet.getComputerNames(), hasSize(2));
+        assertThat(ComputerSet.getComputerNames(), containsInAnyOrder("aNode", "anAnotherNode"));
```

Comment:
`containsInAnyOrder` actually check if the collections are of the same size and contain same elements regardless of their position in the collection. IOW, `hasSize` should not be needed as long as we compare using `containsInAnyOrder`, which I recommend in both cases here.